plugins/examples/custom-suggester/src/main/java/org/elasticsearch/example/customsuggester/CustomSuggestionBuilder.java

```diff
@@ -9,6 +9,7 @@
 package org.elasticsearch.example.customsuggester;
 
 import org.elasticsearch.TransportVersion;
```

Comment:
Is this import still needed?

Reply:
yes, it's in the signature of the method:

```
public TransportVersion getMinimalSupportedVersion() {
```